### ISSUE TITLE
The sensor controls dont seem to work. (Samsung Tab 4) also my sensor controls work in other apps!

### ISSUE THREAD

**Issue by Ultmate-Mario** [zhuowei/MCPELauncher#797]

**Comment by zhuowei** [comment]
@Ultmate-Mario Do ModPE scripts work? Sensor controls is implemented with modpe scripts internally.

**Comment by Ultmate-Mario** [comment]
yes! I use mod scripts all the time! when i true on sensor controls the game keeps my steve character facing forward and looking around just makes me look in that drection and when i move around my tablet nothing happens! also, I didnt know that the makers of block lancher would replay! Thats awesome!!

**Comment by zhuowei** [comment]
@Ultmate-Mario Yeah, sensor controlls only control looking right now. How do you think movement should be handled?

**Comment by Ultmate-Mario** [comment]
no! what i sayed was when i move my tablet nothig happens! and i dont know why.

**Comment by zhuowei** [comment]
@Ultmate-Mario If you turn your tablet, your character should also turn. Moving the tablet won't do anything. To move, use the regular touchscreen controls.

**Comment by Ultmate-Mario** [comment]
i know! i can move with the touch button, but when I TURN my tablet it dose nothing! also, turn is when I move!

**Comment by zhuowei** [comment]
@Ultmate-Mario Maybe you need to calibrate the compass of your tablet? https://support.google.com/gmm/answer/6145351?hl=en

**Comment by Ultmate-Mario** [comment]
i know! i can move with the touch button, but when I TURN my tablet it dose nothing! also, turn is what I ment when i said move!

**Comment by Ultmate-Mario** [comment]
i know! i can move with the touch button, but when I TURN my tablet it dose nothing! also, turn is what I ment when i said move!

**Comment by Ultmate-Mario** [comment]
i know! i can move with the touch button, but when I TURN my tablet it dose nothing! also, turn is what I ment when i said move!

**Comment by Ultmate-Mario** [comment]
i know! i can move with the touch button, but when I TURN my tablet it dose nothing! also, turn is what I ment when i said move!

**Comment by Ultmate-Mario** [comment]
i know! i can move with the touch button, but when I TURN my tablet it dose nothing! also, turn is what I ment when i said move!

**Comment by Ultmate-Mario** [comment]
i know! i can move with the touch button, but when I TURN my tablet it dose nothing! also, turn is what I ment when i said move!

**Comment by Ultmate-Mario** [comment]
i know! i can move with the touch button, but when I TURN my tablet it dose nothing! also, turn is what I ment when i said move!

**Comment by Ultmate-Mario** [comment]
i know! i can move with the touch button, but when I TURN my tablet it dose nothing! also, turn is what I ment when i said move!

**Comment by Ultmate-Mario** [comment]
i know! i can move with the touch button, but when I TURN my tablet it dose nothing! also, turn is what I ment when i said move!

**Comment by Ultmate-Mario** [comment]
i know! i can move with the touch button, but when I TURN my tablet it dose nothing! also, turn is what I ment when i said move!

**Comment by Ultmate-Mario** [comment]
i know! i can move with the touch button, but when I TURN my tablet it dose nothing! also, turn is what I ment when i said move!

**Comment by Ultmate-Mario** [comment]
i know! i can move with the touch button, but when I TURN my tablet it dose nothing! also, turn is what I ment when i said move!

**Comment by Ultmate-Mario** [comment]
i know! i can move with the touch button, but when I TURN my tablet it dose nothing! also, turn is what I ment when i said move!

**Comment by Ultmate-Mario** [comment]
i know! i can move with the touch button, but when I TURN my tablet it dose nothing! also, turn is what I ment when i said move!

**Comment by Ultmate-Mario** [comment]
i know! i can move with the touch button, but when I TURN my tablet it dose nothing! also, turn is what I ment when i said move!

**Comment by Ultmate-Mario** [comment]
i know! i can move with the touch button, but when I TURN my tablet it dose nothing! also, turn is what I ment when i said move!

**Comment by Ultmate-Mario** [comment]
i know! i can move with the touch button, but when I TURN my tablet it dose nothing! also, turn is what I ment when i said move!

**Comment by Ultmate-Mario** [comment]
i know! i can move with the touch button, but when I TURN my tablet it dose nothing! also, turn is what I ment when i said move!

**Comment by Ultmate-Mario** [comment]
i know! i can move with the touch button, but when I TURN my tablet it dose nothing! also, turn is what I ment when i said move!

**Comment by Ultmate-Mario** [comment]
i know! i can move with the touch button, but when I TURN my tablet it dose nothing! also, turn is what I ment when i said move!

**Comment by Ultmate-Mario** [comment]
i know! i can move with the touch button, but when I TURN my tablet it dose nothing! also, turn is what I ment when i said move!

**Comment by Ultmate-Mario** [comment]
i know! i can move with the touch button, but when I TURN my tablet it dose nothing! also, turn is what I ment when i said move!

**Comment by Ultmate-Mario** [comment]
i know! i can move with the touch button, but when I TURN my tablet it dose nothing! also, turn is what I ment when i said move!

**Comment by Ultmate-Mario** [comment]
i know! i can move with the touch button, but when I TURN my tablet it dose nothing! also, turn is what I ment when i said move!

**Comment by Ultmate-Mario** [comment]
i know! i can move with the touch button, but when I TURN my tablet it dose nothing! also, turn is what I ment when i said move!

**Comment by Ultmate-Mario** [comment]
sorry i didn't mean to do 20 posts... anyway, i said that motion my controls work in other games like doodle jump and classic boy. (←its an emulator) so if its working for you, then sensor controls probably dont work for a samsung tablet 4...

**Comment by zhuowei** [comment]
@Ultmate-Mario I tested it just now on my Nexus 6P and I needed to calibrate my compass in Google Maps before it would work.

**Comment by Ultmate-Mario** [comment]
my tablet is changing but i will try and calibrate my compass tomorrow! ill tell you if i works. also, this kind of random but, im making my a ModPE script and texture pack!

**Comment by Ultmate-Mario** [comment]
DARN IT!! i ment to say charging and not changing! DX im not good at touch screens...

**Comment by Ultmate-Mario** [comment]
How do i calabrate my compass? i tryed the link you gave me and tryed it for a little bit. but it did not work.
